compiler/src/iree/compiler/Codegen/LLVMGPU/test/test_query_mma.mlir

```diff
@@ -25,18 +22,65 @@ module {
   }
 }
 
-// CHECK: Printing MMA Collection Before querying supported mma instrinsic instructions, size: 0
-// CHECK: Printing MMA Collection After querying supported mma instrinsic instructions, size: 9
-// CHECK: MFMA_F32_16x16x4_F32
+// CHECK:       Executable Variant Name
+// CHECK-SAME:  main
+// CHECK: MMA   Intrinsics
+// CHECK-SAME:  MFMA_F32_16x16x4_F32
+// CHECK-SAME:  MFMA_F32_16x16x16_F16
+// CHECK-LABEL: func.func @fn
+
+// -----
+
+#executable_target_rocm_hsaco_fb0 = #hal.executable.target<"rocm", "rocm-hsaco-fb",
+{abi = "hip", iree.gpu.target = #iree_gpu.target<arch = "gfx942", features = "",
+wgp = <compute = int32, storage =  b32,
+subgroup =  shuffle|arithmetic, dot =  dp4xi8toi32,
+mma = [<MFMA_F32_16x16x4_F32>, <MFMA_F32_16x16x16_F16>],
+subgroup_size_choices = [64], max_workgroup_sizes = [1024, 1024, 1024],
+max_thread_count_per_workgroup = 1024, max_workgroup_memory_bytes = 65536,
```

Comment:
could we further trim this down but skipping some of these `wgp` properties out entirely?